packages/bpk-component-swap-button/src/BpkSwapButton.module.scss

```diff
@@ -61,9 +63,4 @@
       border: $bpk-border-size-xl solid $bpk-canvas-contrast-day;
     }
   }
```

Comment:
Testing these changes with dev tools on a real device:

**Height and width removed from outer container**
![Screenshot 2024-02-13 at 10 29 48](https://github.com/Skyscanner/backpack/assets/28352715/69763d4a-c54a-4635-82d3-76e6243db4d9)

**Height and width set on button**
![Screenshot 2024-02-13 at 10 30 02](https://github.com/Skyscanner/backpack/assets/28352715/a58b3406-e8ed-438a-8967-6e10ecc7d425)

**Span with line height of 1rem**
![Screenshot 2024-02-13 at 11 21 29](https://github.com/Skyscanner/backpack/assets/28352715/f7271066-79b2-48d2-89ad-f22ecb686f34)

Reply:
Copying styles over to chrome:

<img width="1226" alt="Screenshot 2024-02-13 at 11 24 27" src="https://github.com/Skyscanner/backpack/assets/28352715/f35963bf-ad94-4090-a96c-ba405372f815">

Reply:
**Running build locally in banana**

![IMG_0857](https://github.com/Skyscanner/backpack/assets/28352715/9c469049-e60b-4403-9e9c-ad0c28eaeb58)

Reply:
**Running build locally in banana**

<img width="227" alt="Screenshot 2024-02-13 at 15 00 45" src="https://github.com/Skyscanner/backpack/assets/28352715/48b4db7f-e380-4dc7-9e94-4503e63dac2f">